geopandas/geodataframe.py

```diff
@@ -822,6 +822,24 @@ def to_crs(self, crs=None, epsg=None, inplace=False):
         if not inplace:
             return df
 
+    def estimate_utm_crs(self, datum_name="WGS 84"):
+        """Returns the estimated UTM CRS based on the bounds of the dataset.
+
+        .. versionadded:: 0.10
+
+        .. note:: Requires pyproj 3+
+
+        Parameters
+        ----------
+        datum_name : str, optional
+            The name of the datum to use in the query. Default is WGS 84.
+
+        Returns
+        -------
+        pyproj.CRS
+        """
```

Comment:
Can we also add an example? The simple one is enough - something like
```py
>>> world = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
>>> germany = world.loc[world.name == "Germany"]
>>> germany.estimate_utm_crs()  # doctest: +SKIP
<add an actual pyproj CRS repr here>
```

Reply:
Sure